app/views/schools/school_targets/expired.html.erb

```diff
@@ -0,0 +1,30 @@
+<% content_for :page_title, 'Results of reducing your energy usage' %>
+
+<%= render "title", school_target: @school_target %>
+
+<div class="row">
+  <div class="col">
+
+    <%= render "prompt_to_set_new_target", school_target: @school_target %>
+
+    <p>
+      Your school set a target to reduce its energy usage between between <strong><%= nice_dates(@school_target.start_date) %></strong> and <strong><%= nice_dates(@school_target.target_date) %></strong>. This page summarises your results.
+    </p>
+
+    <% if @progress_summary.present? %>
+      <% if @progress_summary.any_passing_targets?(check_recent: false) %>
+      Well done! You achieved your goal to reduce your <%= t_fuels_as_sentence(@progress_summary.passing_fuel_targets(check_recent: false)) %> usage.
+      <% end %>
```

Comment:
would a safe nav operator `&.` work here (and below)? <% if @progress_summary&.any_passing_targets?(check_recent: false) %> if yes you could remove the ` <% if @progress_summary.present? %>` line above?